.gitlab-ci.yml

```diff
@@ -221,6 +221,7 @@ documentation-assets:gource:
     - docker run -v "$PWD":/visualization $CI_REGISTRY/r3/docker/gource
   artifacts:
     paths: ['output.gif']
+    expire_in: 3 mos
```

Comment:
I'd increase even to 1 year or longer

Reply:
you mean 12 mos

Reply:
I hope that it can parse "year" (the doc is not very authoritative about what it can actually do :D). If not, we go to Mo's.